lib/core/src/rodsPath.cpp

```diff
@@ -555,6 +555,39 @@ clearRodsPath( rodsPath_t *rodsPath ) {
     return;
 }
 
+auto clearRodsPathInp( rodsPathInp_t* path ) -> void {
+  if (nullptr == path) {
+    return;
+  }
+
+  for (int i{}; i < path->numSrc; i++) {
+    clearRodsPath(&path->srcPath[i]);
+    clearRodsPath(&path->targPath[i]);
```

Comment:
So `targPath` is an array?

Reply:
If I understand the struct correctly, every `srcPath` has a paired `targPath`, which is probably based off the `basePath`. The last part is a guess, but I did see `srcPath` and `targPath` paired together, so `numSrc` should represent the length for both.